pymc/model/core.py

```diff
@@ -1233,16 +1162,16 @@ def set_data(
 
     def register_rv(
         self,
-        rv_var,
-        name,
+        rv_var: RandomVariable,
```

Comment:
This is wrong, RandomVariable is not a variable type, it's an Op type

Reply:
Hello there 😊